packages/playwright-test/src/runner.ts

```diff
@@ -272,25 +273,25 @@ export class Runner {
       preprocessRoot._addSuite(fileSuite);
     }
 
-    // 2. Filter tests to respect line/column filter.
+    // 2. Complain about duplicate titles.
+    const duplicateTitlesError = createDuplicateTitlesError(config, preprocessRoot);
+    if (duplicateTitlesError)
+      fatalErrors.push(duplicateTitlesError);
+
+    // 3. Filter tests to respect line/column filter.
     filterByFocusedLine(preprocessRoot, testFileReFilters);
 
-    // 3. Complain about only.
+    // 4. Complain about only.
```

Comment:
Why do you check for `only` after the filtration, but check dupes before the filtration?